api/models.py

```diff
@@ -259,7 +259,7 @@ def translate_fields(cls, params: dict):
         """
         translated = params.copy()
         parent = params.get('parent')
-        if parent:
+        if parent and parent != 'null':
```

Comment:
Same comment as before, I think `null` should have been translated to `None` before reaching this point.